src/app/credExplorer/WeightConfig.js

```diff
@@ -65,26 +62,49 @@ export class WeightConfig extends React.Component<Props, State> {
               justifyContent: "space-between",
             }}
           >
-            <EdgeConfig
-              edgeWeights={this.state.edgeWeights}
-              onChange={(ew) =>
-                this.setState({edgeWeights: ew}, () => this.fire())
-              }
-            />
-            <NodeConfig
-              nodeWeights={this.state.nodeWeights}
-              onChange={(nw) =>
-                this.setState({nodeWeights: nw}, () => this.fire())
-              }
-            />
+            {this.pluginWeightConfigs()}
           </div>
         )}
       </React.Fragment>
     );
   }
 
+  pluginWeightConfigs() {
+    return this.props.adapters
+      .adapters()
+      .filter((x) => x.name() !== FALLBACK_NAME)
```

Comment:
Can we filter by the prefix instead of the name? The former is
_required_ to be unique, while the latter merely _should_ be.

Reply:
Name is required unique.
https://github.com/sourcecred/sourcecred/blob/master/src/app/adapters/adapterSet.js#L34

Reply:
Great; that works, too. Thanks!